pyperf/_metadata.py

```diff
@@ -62,6 +68,7 @@ def format_noop(value):
 LOOPS = _MetadataInfo(format_number, (int,), is_strictly_positive, 'integer')
 WARMUPS = _MetadataInfo(format_number, (int,), is_positive, 'integer')
 SECONDS = _MetadataInfo(format_seconds, NUMBER_TYPES, is_positive, 'second')
+TAGS = _MetadataInfo(format_generic, (tuple, list), is_tags, 'tag')
```

Comment:
It's annoying to have to deal with two different types, tuple or list. I would prefer to always use the same type. A tuple should be good for tags, no?